content/en/platform/corda/4.7/enterprise/node/deploy/running-a-node.md

```diff
@@ -42,10 +42,10 @@ By default, the node will look for a configuration file called `node.conf` and a
 in the current working directory. You can override the configuration file and workspace paths on the command line (e.g.
```

Comment:
Header links that don't resolve:

- running-a-node.html#starting-an-individual-corda-node
- running-a-node.html#starting-all-nodes-at-once